src/main/kotlin/com/jetbrains/snakecharm/lang/psi/SmkFileReference.kt

```diff
@@ -11,21 +11,24 @@ import com.jetbrains.python.psi.PyStringLiteralExpression
 class SmkFileReference(
         element: PsiElement, textRange: TextRange
 ) : PsiReferenceBase<PsiElement>(element, textRange) {
```

Comment:
Here `PsiReferenceBase<PsiElement>` seems to be to general, I think I want `PsiReferenceBase< SMKWorkflowParameterListStatement>`